visualizer/config.py

```diff
@@ -76,28 +76,32 @@ class Config:
     def NUMBER_OF_FRAMES_PER_TURN(self) -> int:
         """
         If you have an animation, this will be the number of frames the animation goes through for each turn.
+        :return: int
         """
         return self.__NUMBER_OF_FRAMES_PER_TURN
 
     @property
     def TILE_SIZE(self) -> int:
         """
-        This will be the size of the tile-its going to be squares
+        This will be the size of the tile-its going to be squares.
+        :return: int
         """
         return self.__TILE_SIZE
 
     @property
     def SCALE(self) -> int:
         """
         Scale is for the tile size being scaled larger.
-        For example: With a 16x16 tile, it can be scaled by 4 to be a 64x64 tile on the visualizer
+        For example: With a 16x16 tile, it can be scaled by 4 to be a 64x64 tile on the visualizer.
+        :return: int
         """
         return self.__SCALE
 
     @property
     def SCREEN_SIZE(self) -> Vector:
         """
-        The screen size is the overall screen size
+        The screen size is the overall screen size.
```

Comment:
Add the following to this: "The x and y values of the Vector object returned represent the horizontal and vertical lengths of the screen respectively."